docs/source/write_first_app.rst

```diff
@@ -1006,8 +598,7 @@ Additional resources
 
 As we said in the introduction, we have a whole section on
```

Comment:
I'll want to rewrite this one. A lot of words saying less.

Reply:
Removed.